### PR TITLE
updating step definitions

### PR DESCRIPTION
**Pull request by LokeshSrinivasa**

### CODE REVIEW

File: src/test/resources/features/GetBSPeriods.feature [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#  Scenario:
+#    Given A valid user request for Identity Endpoint "/individuals/breathing-space/debtor/MZ006526C/details"
+#    When I add given headercase "ValidHeaders"
+#    When I submit the "GET" request for
+#
+#  Scenario 2:
+#    Given A valid user request for BS_Period Endpoint "/individuals/breathing-space/MZ006526C"
+#    When I submit the "GET" request for
```
Comment (glidester) [inline src/test/resources/features/GetBSPeriods.feature:8]: Missing blank line

File: src/test/resources/app.conf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+dclientId  = "zhErAFbY75DCa0cpls7uT0wLaT4g"
+dappclient = "0f7ab028-69fd-44d4-9e17-ffce572cc3c6"
+dtotp = "HIGG2EK5IV4A2ISI"
+qclientId  = "5EjVt3x1euTtw5Rw6AY3vEghkmOr"
+qappclient = "4599a6e2-d7f9-4a95-ae38-3ec9fff39245"
+qtotp = "KQUUKWIWEISQEQKF"
+sclientId  = "Vwj6CafO7sC6LZSzPzubLLeRw0lA"
+sappclient = "23ba4d45-b982-4be9-9459-a741aad3571b"
+stotp = "INGR2RIBLE5U2JJJ"
```
Comment (glidester) [inline src/test/resources/app.conf:9]: Is it safe to put these secret keys into the git hub repo?????

File: src/test/scala/uk/gov/hmrc/stepdefs/testAPIStepDefs.scala [hunk 3]
```diff
@@ -1,65 +1,91 @@
...
-  val nino: String = "MZ006526C"
-  val getIdentityEndpoint: String = s"/individuals/breathing-space/debtor/$nino/details"
-
   val requestMap: scala.collection.mutable.Map[String, Object] = scala.collection.mutable.Map[String, Object]()
-
-  //var ENDPOINT_URL :String = Configuration.settings.Url.toString + getIdentityEndpoint
   var ENDPOINT_URL: String = ""
   val REQUEST_HEADERS: String = "headers"
-  val REQUEST_PAYLOAD: String = "payload"
+  var REQUEST_PAYLOAD: String = "payload"
```
Comment (glidester) [inline src/test/scala/uk/gov/hmrc/stepdefs/testAPIStepDefs.scala:16]: Is this mutability done on purpose or is it a mistake?

File: src/test/scala/uk/gov/hmrc/utils/TokenGenerator.scala [hunk 4]
```diff
@@ -5,10 +5,10 @@ import scalaj.http.Http
 
 object TokenGenerator {
 
-  val clientId = Configuration.settings.clientId
-  val appclientSecret = Configuration.settings.appclientSecret
-  val totpSecret = Configuration.settings.totpSecret
-  val oauthFrontendApiBaseUrl = Configuration.settings.oauthFrontendApiBaseUrl
+  var clientId: String = Configuration.settings.clientId
```
Comment (glidester) [inline src/test/scala/uk/gov/hmrc/utils/TokenGenerator.scala:8]: Mutable on purpose?

**Review by glidester — CHANGES_REQUESTED** [review]

File: src/test/resources/app.conf (hunk 2 above)
Comment (LokeshSrinivasa) [reply src/test/resources/app.conf:9]: i have seen other projects have them on github. is there a way someone to confirm?

File: src/test/scala/uk/gov/hmrc/stepdefs/testAPIStepDefs.scala (hunk 3 above)
Comment (LokeshSrinivasa) [reply src/test/scala/uk/gov/hmrc/stepdefs/testAPIStepDefs.scala:16]: this was on purpose there is a case statement to get payload later in the code.

File: src/test/scala/uk/gov/hmrc/utils/TokenGenerator.scala (hunk 4 above)
Comment (LokeshSrinivasa) [reply src/test/scala/uk/gov/hmrc/utils/TokenGenerator.scala]: corrected it now

File: src/test/resources/features/GetBSPeriods.feature (hunk 1 above)
Comment (LokeshSrinivasa) [reply src/test/resources/features/GetBSPeriods.feature:8]: this isn't started yet.

**Comment by LokeshSrinivasa** [comment]
not required

**Pull request closed without merging**